src/Console/AlgoliaCommand.php

```diff
@@ -2,25 +2,29 @@
 
 namespace Algolia\Settings\Console;
 
+use Algolia\Settings\Services\Resource as ResourceService;
```

Comment:
I hope we can come up with a more explicite name, but I don't have much inspiration.